src/spikeinterface/core/base.py

```diff
@@ -561,23 +564,28 @@ def dump(self, file_path: Union[str, Path], relative_to=None, folder_metadata=No
     def dump_to_json(self, file_path: Union[str, Path, None] = None, relative_to=None, folder_metadata=None) -> None:
         """
         Dump recording extractor to json file.
-        The extractor can be re-loaded with load_extractor_from_json(json_file)
+        The extractor can be re-loaded with load_extractor(json_file)
 
         Parameters
         ----------
         file_path: str
             Path of the json file
-        relative_to: str, Path, True or None
-            If not None, files and folders are serialized relative to this path. If True, the relative folder is the parent folder.
-            This means that file and folder paths in extractor objects kwargs are changed to be relative rather than absolute.
+        relative_to: str, Path, or None
```

Comment:
Why did you modify the docstring to the old typing?

Reply:
mistake in merging, fixing back